### ISSUE TITLE
Nvda does not read the name of the text boxes displayed in the account creation dialog in GitHub,

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7014]
Using nvda next 13946 and chrome 57, just by pressing the new issue button in GitHub, a dialog is displayed to create a new account.
When walking with tabe through this dialog titles like name, email, password are not read by nvda.
I usually read the name of these text fields by pressing nvda + tab, but in this way nvda does not read the title of text boxes.
Follow the log:

IO - inputCore.InputManager.executeGesture (09:54:54):
Input: kb(laptop):tab
IO - speech.speak (09:54:54):
Speaking [u'lista com 2 itens', u'edi\xe7\xe3o', u'Join GitHub  di\xe1logo']
IO - inputCore.InputManager.executeGesture (09:55:01):
Input: kb(laptop):NVDA+tab
IO - speech.speak (09:55:01):
Speaking [u'edi\xe7\xe3o  com foco']
IO - speech.speak (09:55:01):
Speaking [u'em branco']
IO - inputCore.InputManager.executeGesture (09:55:08):
Input: kb(laptop):NVDA+f1
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (09:55:08):
Developer info for navigator object:
name: None
role: ROLE_UNKNOWN
states: 
isFocusable: False
hasFocus: False
Python object: <NVDAObjects.IAccessible.IAccessible object at 0x0BC4E590>
Python class mro: (<class 'NVDAObjects.IAccessible.IAccessible'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: None
location: None
value: None
appModule: <'appModuleHandler' (appName u'chrome', process ID 3036) at address 5e83ed0>
appModule.productName: u'Google Chrome'
appModule.productVersion: u'56.0.2924.87'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 918518
windowClassName: u'Chrome_RenderWidgetHostHWND'
windowControlID: 174919104
windowStyle: 1445986304
windowThreadID: 3212
windowText: u'Chrome Legacy Window'
displayText: exception: 'NoneType' object is not iterable
IAccessibleObject: <POINTER(IAccessible) ptr=0xcbf0ab0 at bba88a0>
IAccessibleChildID: 0
IAccessible event parameters: windowHandle=918518, objectID=-4, childID=-13033
IAccessible accName: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accRole: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accState: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accDescription: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))
IAccessible accValue: exception: (-2147417842, 'O aplicativo chamou uma interface marshalled para um outro thread.', (None, None, None, 0, None))

**Comment by jcsteh** [comment]
This is a problem with the GitHub website. The labels are not associated with the controls correctly.
